steve-chae-multiple-choice-questions.md

```diff
@@ -0,0 +1,96 @@
+<!--
+	Correct answer is 2.
+	
+	1 - NgModule is the name of the Decorator used for any Angular Module.
+	I feel one can easily mistake between NgModule and AppModule
+	
+	3 - while it's possible that the root module can be anything, 
+	convention is to name it AppModule and that's what our slides say. 
+	This could be a good chance to bring it up that it's convention.
+	
+	(https://angular.io/docs/ts/latest/guide/appmodule.html)
+-->
+The name for the root module should be _________.:
+
+1. `NgModule`
+1. `AppModule`
+1. anything
+
+---
+
+<!--
+	Correct answer is 1.
+	
+	The root module is used exactly for bootstrapping and the boostrap 
+	property is specially available to the root module
+	
+	2, 3, 4 - are also available to other modules and not really special.
+	If students pick this maybe we didn't stress the purpose of the root
+	module strong enough
+	 
+
+-->
+Which of the following property is special to the root module?
+
+1. `bootstrap`
+1. `declarations`
+1. `providers`
+1. None. The root module is just like any other modules
```

Comment:
Perhaps we can have one of the wrong answers as 'exports' since that is special to only `feature modules` and shouldn't exist on `root module` at all